virtualizarr/tests/__init__.py

```diff
@@ -42,6 +44,11 @@ def _importorskip(
 has_zarr_python, requires_zarr_python = _importorskip("zarr")
 has_zarr_python_v3, requires_zarr_python_v3 = _importorskip("zarr", "3.0.0b")
 
+hdf_backend = pytest.mark.parametrize(
+    "hdf_backend",
+    [HDF5VirtualBackend, HDFVirtualBackend] if has_kerchunk else [HDFVirtualBackend],
+)
```

Comment:
Oh that's really neat, nice.

Reply:
Maybe just change the name to something like `parametrize_over_hdf_backends` to make it more obvious what this decorator does.